examples/lightgbm/pixi.toml

```
@@ -0,0 +1,14 @@
+[project]
+name = "lightgbm"
+version = "0.1.0"
+description = "LightGBM demo for machine learning project"
+authors = ["fkt <deepthought@postech.ac.kr>"]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
+
+[tasks]
```

```suggestion
[tasks]
start = "python main.py"
```
We try to always add a `start` command :smile: